ui/src/pages/session-details/session-details.tsx

```diff
@@ -46,6 +47,9 @@ export const SessionDetails = () => {
 
   return (
     <div className={styles.main}>
+      <Helmet>
+        <meta name="og:image" content={session.exampleCaptures[0]?.src} />
```

Comment:
Is `exampleCaptures` available in the session detail response? I don't see it. Can we use the current capture image instead? With a fallback to `first_capture` of the whole session?